Library/JsonConfig.ts

```diff
@@ -18,6 +18,7 @@ const ENV_noDiagnosticChannel = "APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL"
 const ENV_noStatsbeat = "APPLICATION_INSIGHTS_NO_STATSBEAT";
 const ENV_noHttpAgentKeepAlive = "APPLICATION_INSIGHTS_NO_HTTP_AGENT_KEEP_ALIVE";
 const ENV_noPatchModules = "APPLICATION_INSIGHTS_NO_PATCH_MODULES";
+const ENV_webSnippetEnable = "APPINSIGHTS_WEB_SNIPPET_ENABLED";
```

Comment:
We have "APPLICATION_INSIGHTS", "APPLICATIONINSIGHTS" and now APPINSIGHTS, maybe stick to some of the other two, I recommend to use "APPLICATIONINSIGHTS_WEB_SNIPPET_ENABLED"